text/0000-arbitrary-self-types-v2.md

```diff
@@ -93,20 +114,7 @@ pub trait Receiver {
 }
 ```
 
-A blanket implementation is provided for any type that implements `Deref`:
-
-```rust
-impl<P: ?Sized, T: ?Sized> Receiver for P
-where
-    P: Deref<Target = T>,
-{
-    type Target = T;
-}
-```
-
-(See [alternatives](#no-blanket-implementation) for discussion of the tradeoffs here.)
-
-It is also implemented for `&T`, `&mut T`, `*const T` and `*mut T`.
+Implementations are provided for all standard library types which implement `Deref`. It is also implemented for `&T`, `&mut T`, `*const T` and `*mut T`.
```

Comment:
Do we have to do it for all linstd types that implement Deref? For the existing stable Self types (Pin, Box, ...) would be enough, wouldn't it?